src/components/VtkTwoView.vue

```diff
@@ -47,6 +47,37 @@ function lpsDirToLabels(dir) {
   return label;
 }
 
+const createGeometryJob = (rep, sliceRange, view) => ({
+  name: `Geometry Slice Job for ${rep.getProxyId()}`,
+  // TODO rep and view could be deleted by this point
+  entry: (done, isCancelled) => {
+    // generate slice info
+    const axis = view.getAxis();
+    const normal = [0, 0, 0];
+    normal[axis] = 1;
+    const pos = [0, 0, 0];
+    const slices = [];
+    for (let s = sliceRange[0]; s < sliceRange[1]; s += 1) {
+      slices.push(s);
+    }
+
+    let i = 0;
+    // TODO do this in a web worker
```

Comment:
Is converting to a web worker a lot of work? If so, I would remove this comment for now and outline the work in an issue instead.